apps/designer/app/routes/canvas/$projectId.$pageId.tsx

```diff
@@ -1,2 +0,0 @@
-export * from "./$projectId";
-export { default } from "./$projectId";
```

Comment:
This was a bad idea indeed. This confuses Remix somehow so it doesn't load data when you navigate in browser